xee/ext.py

```diff
@@ -186,6 +191,10 @@ def __init__(
       ee_init_kwargs: Optional[Dict[str, Any]] = None,
       ee_init_if_necessary: bool = False,
       executor_kwargs: Optional[Dict[str, Any]] = None,
+      tile_fetch_kwargs: Dict[str, int] = {
```

Comment:
Lets define a constant dict `TILE_FETCH_KWARGS` at the top.

```
 TILE_FETCH_KWARGS = {
          'max_retries': 6,
          'initial_delay': 500,
 }
 ```
 
 And make this argument optional here.

Reply:
I don't think we need to convert this argument as a optional argument as I already passed the default value here.

Reply:
check my another comment.